src/modules/FilteredLearningModule.js

```diff
@@ -10,6 +10,7 @@ function FilteredLearningModule(props) {
     <LearningModuleListItem key={metaDatum.id} moduleMetadata={metaDatum} />
   ));
 
+  
```

Comment:
I saw this and it reminded me of a time where I decided to change some code formatting in a bunch of places, along with my new code. I adding some spaces, and new lines here and there, things like that. My lead at time time let me know that lots of little changes like that add more files to the PR's, and that they can distract from the actual code changes. -- You only did it once and is really not worth commenting on, but I figured I'd take the opportunity to share the experience.

Reply:
@pcrain9 testing pings.

Reply:
@CSegotaDev testing!